source/includes/steps-open-new-playground.yaml

```diff
@@ -21,14 +21,13 @@ content: |
    :guilabel:`MongoDB:`.
 
    When you run the :guilabel:`MongoDB: Create MongoDB Playground`
-   command, |vsce| opens a playground pre-configured with a few
-   commands.
+   command, |vsce| opens a default playground template pre-configured
+   with a few commands.
 
    .. note::
 
-      You can load new Playgrounds without the template by
-      disabling the :guilabel:`Use Default Template For Playground`
-      setting. To learn more about |vsce| settings, see
-      :ref:`vsce-settings`.
+      To load new Playgrounds without the template, disable the
+      :guilabel:`Use Default Template For Playground` setting. To learn
```

Comment:
(Throughout) :guilabel:`Use Default Template For Playground` doesn't match the normal rendering of a parameter. It's not the same as for an internal parameter. Updating it across these docs would be too much scope creep, but the rendering seems really inconsistent.

Reply:
That is a fair comment, but we have typically used `guilabel` for settings found in a UI, rather than specified in code. That's the distinction we're making.

Reply:
Done 

Is that our choice or something passed over to us from the design team? We should think about changing it since the current style looks like we've messed up our markup.